plugin/notes/notes.html

```diff
@@ -559,7 +562,17 @@ <h4 class="label">Notes</h4>
 								}
 								timings.push(timing);
 							}
-
+							if ( totalTime ) {
+								// After we've allocated time to individual slides, we summarize it and
+								// subtract it from the total time
+								var remainingTime = totalTime - timings.reduce( function(a, b) { return a + b; }, 0 );
```

Comment:
Note to self: this can be negative, resulting in timePerSlide being negative for all slides as well. Not good. Will fix.

Reply:
First stab at a fix for this is in 5bf78d2967b1706dbfda06d27512c13fc4f12048.